documentation/source/conf.py

```diff
@@ -95,7 +96,11 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = []
+exclude_patterns = [
+    "newsfragments/*.rst",
+    "generated/namespacelist.rst",
+    "generated/namespace/*.rst",
```

Comment:
If you know where these come from, can you add a comment?

Reply:
I think this is correct?
```suggestion
    # unused outputs from breathe
    "generated/namespacelist.rst",
    "generated/namespace/*.rst",
```

Reply:
It is, thanks.